lib/cisco_node_utils/bgp.rb

```diff
@@ -368,28 +353,19 @@ def default_confederation_peers
 
     # Graceful Restart Getters
     def graceful_restart
-      match = config_get('bgp', 'graceful_restart', @get_args)
-      match.nil? ? false : true
+      config_get('bgp', 'graceful_restart', @get_args)
```

Comment:
Yup. In general, if we have a command that NVGENs its 'no' form (because true, not false, is default), then having the token match the 'no' form is key to getting the correct automatic behavior.